x-pack/plugins/alerting/server/task_runner/rule_action_helper.ts

```diff
@@ -102,6 +116,7 @@ export const getSummaryActionTimeBounds = (
   if (!isSummaryAction(action)) {
     return { start: undefined, end: undefined };
```

Comment:
Should we return `getSummarySystemActionTimeBounds`? Let's talk about it offline. It seems that we need to decide how to handle the system action time bounds.